packages/calypso-e2e/src/lib/blocks/block-flows/pay-with-paypal.ts

```diff
@@ -0,0 +1,65 @@
+import { BlockFlow, EditorContext, PublishedPostContext } from '..';
+import { GutenbergEditorPage } from '../..';
+
+interface ConfigurationData {
+	name: string;
+	price: number;
+	email: string;
+}
+
+const blockParentSelector = 'div[aria-label="Block: Pay with PayPal"]';
+const selectors = {
+	// Editor
+	name: 'input[placeholder="Item name"]',
+	currency: '.simple-payments__field-currency .components-select-control__input',
+	price: '.simple-payments__field-price .components-text-control__input',
+	email: 'input[placeholder="Email"]',
```

Comment:
Optional suggestion: I could see inputs with a placeholder of "Email" being something that might not be unique to this block (I'm thinking of our many form-based blocks).

What do you think of preceding this selector with the parent block selector for a little extra safety when we are adding multiple blocks to a post?

```typescript
email: `${ blockParentSelector } input[placeholder="Email"]`,
```